build_tools/travis/install.sh

```diff
@@ -112,6 +114,7 @@ else
     python --version
     python -c "import numpy; print('numpy %s' % numpy.__version__)"
     python -c "import scipy; print('scipy %s' % scipy.__version__)"
+    python -c "import pandas; print('pandas %s' % pandas.__version__)" 
```

Comment:
it would be better if this didn't show an ImportError traceback if pandas is not available.